packages/@aws-cdk/cloudformation-diff/lib/diff-template.ts

```diff
@@ -184,3 +208,98 @@ function deepCopy(x: any): any {
 
   return x;
 }
+
+function filterFalsePositivies(diff: types.TemplateDiff, changeSet: CloudFormation.DescribeChangeSetOutput) {
+  const replacements = findResourceReplacements(changeSet);
```

Comment:
Looks like for every change we are going to see if it's a replacement or not. 

What happens to additions and deletions?

Reply:
Our schema-based diff is always correct about resource additions / deletions, so I didn't check the change set for those.

Reply:
Do you have a test on that?